plugin/storage/integration/elasticsearch_test.go

```diff
@@ -96,6 +96,8 @@ func (s *ESStorageIntegration) initializeES(allTagsAsFields, archive bool) error
 	s.esCleanUp(allTagsAsFields, archive)
 	// TODO: remove this flag after ES support returning spanKind when get operations
 	s.notSupportSpanKindWithOperation = true
+
+	s.Fixtures = loadAndParseQueryTestCases(t, "fixtures/queries_es.json")
```

Comment:
It's better to load them in `testElasticsearchStorage`. This func initializes the storage.